src/app/app-interactive-map/app-interactive-map.component.ts

```diff
@@ -73,10 +76,7 @@ export class AppInteractiveMapComponent implements OnInit, AfterViewInit {
   }
 
   ngOnInit(): void {
-    this.store.dispatch(new fromActions.FetchSpecies());
-    this.store.dispatch(new fromActions.FetchMaps());
-    this.store.dispatch(new fromActions.FetchModels());
-
+    this.store.dispatch(new FetchModels());
```

Comment:
I dont know why if I removed it, when you are going to the interactive map from the sidebar, we are not drawing the map, I was looking for the error the whole evening but I could not find it.